gt-elasticsearch/src/main/java/mil/nga/giat/data/elasticsearch/ElasticDataStoreFactory.java

```diff
@@ -136,6 +137,15 @@
             GRID_THRESHOLD
     };
 
+    /**
+	 * The ES RestClient instances created by this factory. We don't want to create
```

Comment:
Can unindent these comment lines by two spaces for consistency with other formatting elsewhere